tests/dummy/app/templates/ember-flexberry-dummy-suggestion-edit.hbs

```diff
@@ -131,17 +132,15 @@
       limitFunction=null
       class=(if model.errors.author "error" "")
     }}
+    {{flexberry-validationmessage error=(t "forms.ember-flexberry-dummy-suggestion-edit.vauthor-caption") pointing="pointing"}}
   </div>
-  <div class="field">
+  <div class="field {{if model.errors.editor1 "error" ""}}">
     <label>{{t "forms.ember-flexberry-dummy-suggestion-edit.editor-caption"}}</label>
-    {{#if model.errors.editor}}
-      <span style="color:red">{{model.errors.editor}}</span>
-    {{/if}}
     {{flexberry-lookup
       value=model.editor1
       projection="ApplicationUserL"
       displayAttributeName="name"
-      title="Editor"
+      title="Editor1"
```

Comment:
Here title=Editor is just user friendly name for "belongsTo" relationship,
i'm not sure that it was necessary to rename it to model's property name "Editor1",
but if you  think it is simplier, ok then.